src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -399,7 +399,7 @@ void clocks_enable_resus(resus_callback_t resus_callback);
 /*! \brief Output an optionally divided clock to the specified gpio pin.
  *  \ingroup hardware_clocks
  *
- * \param gpio The GPIO pin to output the clock to. Valid GPIOs are: 21, 23, 24, 25. These GPIOs are connected to the GPOUT0-3 clock generators.
+ * \param gpio The GPIO pin to output the clock to. Valid GPIOs are: 21 (or 13 on RP2350), 23 (or 15 on RP2350), 24, 25. These GPIOs are connected to the GPOUT0-3 clock generators.
```

Comment:
i think this is a bit confusing - could this read as "21 on RP2040 OR 13 on RP2350" possibly "and" or "and/or"
also maybe ", and 25"